cockpit/src/subscriptions.css

```diff
@@ -41,6 +54,12 @@
     padding-left: 20px;
 }
 
+
+#register_dialog .modal-footer .dialog-wait-ct .spinner {
```

Comment:
I think this is no longer necessary after #2196, or is it?

Reply:
Yeah, it can be removed ATM (after merge from master). But it is necessary to fix CSS, because error message in registration dialog looks like this:

![image](https://user-images.githubusercontent.com/2057012/71164923-2f07f480-2250-11ea-8e76-c51ab7d5112c.png)